dev/com.ibm.websphere.appserver.features/visibility/public/crac-1.0/resources/l10n/io.openliberty.crac-1.0.properties

```
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the Coordinated Restore at Checkpoint org.crac APIs.
```

per IBM style, put code entity before noun identifiers
```suggestion
description=This feature enables the org.crac Coordinated Restore at Checkpoint  APIs.
```

---

I'll update this, but I question sending messages off to translation for noship features at this time.